Documentation/KnownIssues/Template/NETFXKnownIssuesTemplate.md

```diff
@@ -4,7 +4,7 @@
 // Entries with [| ... |] are fields that must be updated.
 
 ## Symptoms
-[|Description of the symtopms|]
+[|Description of the symtoms|]
```

Comment:
It should be "symptom".

Reply:
Shouldn't do corrections without sleep. I'll close this and create a new PR. Ty for pointing this out.